src/Atoms/UserProfileData/UserProfileData.js

```diff
@@ -0,0 +1,23 @@
+import propTypes from 'prop-types';
+
+import UserProfileDataStyled from './UserProfileDataStyled';
+import participants from '../../Data/MongoDBMock/participants';
+
+const UserProfileData = ({ userData }) => {
+  const userObject = participants.find(
```

Comment:
You shuold not need to find it now, it shuold come in props, prepare it so it does so. This is a dumb component, you should tell it exactly what to do, and all the data it needs should come in props.

Reply:
But in case you need this compoenent to later on fetch any information from the db, prepare it to handle it in useEffect.

Reply:
Thanx for this comment. Fixed